Spanner/src/Batch/BatchSnapshot.php

```diff
@@ -125,6 +125,9 @@ public function close(array $options = [])
      *           each partition may be smaller or larger than this size request.
      *           **Defaults to** `1000000000` (i.e. 1 GiB).
      *     @type string $index The name of an index on the table.
+     *     @type bool $dataBoostEnabled This is an optional parameter which can
+     *           be used for partition read and query to execute the request via
```

Comment:
```suggestion
     *           be used for partition read query to execute the request via
```
here and at other places

Reply:
It can be used in both of the queries.
PartitionRead eventually uses `ReadRequest` and PartitionQuery uses `ExecuteSqlRequest`. Both of them can make use of this feature.